docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -62,6 +62,8 @@
 * [Canal: Alibaba mysql database binlog incremental subscription & consumer components](https://github.com/alibaba/canal) 1.0.25 -> 1.1.2
 * JSON
   * [GSON](https://github.com/google/gson) 2.8.x (Optional²)
+* Customize Enhance
```

Comment:
This don't need to be in this list. This is a feature.

Reply:
Ok